v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
```

Comment:
I was just not familiar enough with these terms. A quick search of `1-indexed` matches the intended meaning here. And "homogenous" is also fine, as long as we think it's clear to users that a array can contain only values of the same non-array type.

For the "1-indexed" aspect, I think we should add another example showing how to get a value out of an array based on its index position. @justinj, perhaps you already have such an example?